compiler/Averagingfilter_SDLT_samples/cpp/Makefile

```diff
@@ -30,11 +30,6 @@ TARGET := $(BUILDDIR)/AveragingFilter
 
 icpc: $(TARGET)
 
-gcc: CXX := g++
```

Comment:
After the merge please test the build to make sure this change doesn't break it.